mkosi

```diff
@@ -1028,7 +1028,7 @@ def install_opensuse(args, workspace, run_build_script):
     #
     # Install the "minimal" package set.
     #
-    subprocess.run(cmdline + ["-t", "pattern", "minimal_base"], check=True)
+    subprocess.run(cmdline + ["patterns-base-minimal_base"], check=True)
```

Comment:
> I reported this issue and it should be fixed soon.

Do you have a link to a bugreport or something else? I don't have the necessary pattern on TW.